costajohnt/passion_project.html

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
```

Comment:
@isomdurm It seems you've taken a disliking toward some of the other students.  Please don't delete all of their work.  

When you delete something from your fork and make a PR with that change, it **will** affect the upstream (i.e. this) repo.  I think we'll need to sit down for a few minutes and sort out your git after class.  Please find me.